chcemvediet/apps/inforequests/tests/test_views/test_create.py

```diff
@@ -237,14 +239,15 @@ def test_post_with_draft_button_and_invalid_data_redraws_form(self):
         self._login_user()
         response = self.client.post(reverse(u'inforequests:create'), data)
         self.assertEqual(response.status_code, 200)
-        self.assertTemplateUsed(response, u'inforequests/create.html')
+        self.assertTemplateUsed(response, u'inforequests/create/create.html')
 
     def test_post_with_draft_button_and_invalid_data_related_models_are_prefetched_before_render(self):
         data = self._create_post_data(button=u'draft', obligee=u'Invalid')
         self._login_user()
-        with self.assertQueriesDuringRender([u'FROM "accounts_profile"']):
+        with self.assertQueriesDuringRender(render_query_patterns.base):
             response = self.client.post(reverse(u'inforequests:create'), data)
 
+    @unittest.skip(u'FIXME')
```

Comment:
@martinmacko47 Sem nastava problem, ze ak sa uklada nova Action, tak sa vytvori kopia Attachmentu, tym padom sa vytvori aj nova instancia a nie je mozne porovnavat staru s novou. Ako to vyriesime, budeme porovnavat ich content? 
```
            ...
            for attch in attachments:
                attachment = attch.clone(action)
                attachment.save()
```

Reply:
Ano, pls porovnaj ich content. Pri ich vyrvarani im vsak daj nejaky explicitny content, ktory potom porovnas. Nech sa nepouzije defaulny content, ktory je vsade rovnaky.